docs/VerifPlans/Simulation/VplanStatusReviews.csv

```diff
@@ -0,0 +1,29 @@
+,,,,,
```

Comment:
In general, looks good. I agree with your reasons for deleting files. I didnt look through the content, only checking that the correct files are generated.

Reply:
> I didnt look through the content, only checking that the correct files are generated (as that is the purpose of this PR (as far as I understood))

Thanks. That's perfect.